_posts/blog/2018-11-02-el-topo-blindado.md

```diff
@@ -0,0 +1,59 @@
+
+I have chosen to also highlight the previous incarnation(s) of this digital archive project precisely because the engagement in political struggle as it pertains to the archive that Weld describes is also a struggle in time. The research group that maintain this site have evolved from seminars, conferences, classes and local meetings, through a blog platform to the digital archive platform being reviewed here. An archive in this sense is in a constant state of becoming rather than existing as a fixed thing. 
+
+For all of the amazing potential of the technology that makes digital archives possible, there are, however, limitations. 
+
+Digital technology requires two things that need to be taken into consideration if the becoming of the archive is to be promoted rather than stalled, as appears to be the case with *El Topo Blindado*: computing design expertise and the time to dedicate oneself to the project. Regarding the first of these, the archive under review, for all of its positive aspects, is rather clunky in appearance and not intuitive in terms of user needs. The site deploys a straightforward dropdown menu system that, at best, manages to double up (and therefore make confusing) the paths one as a user must take to get the information one is looking for; and at worst, leads to empty dead-end sections that provide no information whatsoever to the user regarding what has happened or even the current status of the section: it is simply a dead-end, as is the case with the *Articles* section and dropdown menu.  The site is in need of the loving care only a person with an aptitude for design and a concern for user experience can provide. This will of course cost money and/or resources such as the time taken to learn these skills. 
+
+The second of these issues is the stalling of the project. As far as I can tell there has been no activity on this site for some years now. This is of course unfortunate, it highlights the fact that the technological revolution that makes the almost *infinite becoming* of digital archives possible is not all digital. The digital and the technological, like any and everything else in life is *entangled with life itself*. This means literally that life gets in the way sometimes. Is this not a poignant reflection on the way that conviction and commitment is never the only resources one needs to continue the political struggle in any of its forms? 
+
+Attempts made by me to contact the collective in order to get some sense of how and why the project seems to have stalled - Is it in the process of transforming into another iteration? Has the collective disbanded? Has there been simply no time and/or resources to continue? - have produced no result but silence. 
```

Comment:
This concluding paragraph suggests to me that the most important thing to note about the archive is that the developers have not answered your emails. I wonder if there's anything else you'd like to emphasize here. Do you think the collection is making a valuable contribution to historical research? Does it fill a gap in digital record-keeping? Does it teach us something about digitization more broadly?